provider/cluster/kube/apply.go

```diff
@@ -138,22 +223,44 @@ func prepareEnvironment(ctx context.Context, kc kubernetes.Interface, ns string)
 			},
 		}
 		_, err = kc.CoreV1().Namespaces().Create(ctx, obj, metav1.CreateOptions{})
+		label := "success"
+		if err != nil  {
+			label = "fail"
+		}
+		kubeCallsCounter.WithLabelValues("namespaces-create", label).Inc()
```

Comment:
It'd be nice to put this whole pattern into a function.

```go
func incrementCounterWithLabel(counter Interface,name string,err error) {
   label := "success"
   if err != nil {
    label = "fail"
  }
  counter.WithLabelValues(name,label).Inc()
}
```

Reply:
Yeah, that's a good point. Also it solves some other code organization concerns I had. I'll make a pass to do this.